storage/boltz/query_test.go

```diff
@@ -377,17 +377,36 @@ func (test *boltQueryTests) testBusinessEquals(*testing.T) {
 }
 
 func (test *boltQueryTests) testSortPage(*testing.T) {
-	//paging := &predicate.Paging{Offset: 0, Limit: 10}
-	//sorts := []predicate.SortField{{Field: "lastName", Direction: predicate.DESC}, {Field: "firstName", Direction: predicate.ASC}}
-	ids, count := test.query(`firstName in ["Alice", "Bob", "Cecilia", "David"] SORT BY lastName desc, firstName limit 10`)
```

Comment:
Add test where we have duplication in sort parameters, to ensure we don't drop duplicate sort parameters